ci/docker-deploy.sh

```diff
@@ -13,7 +13,7 @@ docker images
 docker tag feedbacksystem_secrettokenchecker thmmniii/secrettokenchecker:$tag
 docker tag feedbacksystem_ws thmmniii/ws:$tag
 docker tag feedbacksystem_sqlchecker thmmniii/sqlchecker:$tag
-docker tag feedbacksystem_bash thmmniii/bash:$tag
+docker tag feedbacksystem_bash1 thmmniii/bash:$tag
```

Comment:
Why do we use numbers as a suffix in a name? -> e.g., in 'bash1'

Reply:
We already have a "bash" docker, but this is the one, I create, I can name it how ever I want. Do you have a better idea?

Reply:
I understand that you as a developer can name your files like you want.
However, I see that you renamed the feedbacksystem_bash to feedbacksystem_bash1, but push it to thmmniii/bash:$tag. From my perspective, I ask myself: Why do we need to rename it? Why do we not push feedback system_bash to thmmniii/bash:$tag anymore? Moreover, why is it not possible to give it a speaking name, i.e., a name that gives me some hint about the purpose of this docker image. It could be that bash1 is the perfect name for it. Is that the case here?

Reply:
the `bash1` thing comes from the docker-compose file. Indeed, it is a bad name, i.e. not understandable. However, I don't know how to call it. It is a docker image with can run bash and php scripts. So, I don't know, what a good name I can choose.